src/resource/archive/O2rArchive.cpp

```diff
@@ -83,4 +83,4 @@ bool O2rArchive::Close() {
 
     return true;
 }
-} // namespace LUS
+} // namespace LUS
```

Comment:
same thing as config.h

```suggestion
} // namespace LUS

```